### DISCUSSION TITLE
Plans for supporting SPP encoding?

### DISCUSSION THREAD

**Discussion by EmNewman** [CCSDSPy/ccsdspy#94] [Q&A]
Hi there, great work on this library. It looks like currently the library only supports decoding data from the SPP packet protocol. I was wondering if there are any plans to also support encoding data into an SPP packet (generating headers, etc)? Thanks so much!

**Comment by ddasilva** [comment]
Hi @EmNewman, thanks for your kind words. We have encoding implemented on a branch called `packet_encoding` (discussed in #50), and it is ready to be tested, but we've been busy with other things. Would you like to be our first user? 

Adding @ehsteve to the ping list because he originally authored this branch.

**Reply by ddasilva** [reply]
Hi @EmNewman, wanted to ping this thread again. Is this something you're still interested in?
